tests/ssh-hassh/test.yaml

```diff
@@ -0,0 +1,20 @@
+requires:
+  min-version: 5.0.0
+features:
+    - RUST
+
+args:
+ - -k none
+
+checks:
+  # Check that we have the following events in eve.json
+  # 2 - alert events
+  # 1 - tls event
+  - filter:
+      count: 4
```

Comment:
Should we check that there is one alert per signature (ie per sid) ?

Reply:
Yes, will add now.